tests/Integration/Behaviour/Features/Context/Domain/AbstractDomainFeatureContext.php

```diff
@@ -64,9 +64,10 @@ public static function prepare(BeforeSuiteScope $scope)
     public function checkLastException(AfterScenarioScope $scope)
     {
         $e = $this->getLastException();
+        $this->cleanLastException();
 
         if (TestResult::FAILED === $scope->getTestResult()->getResultCode() && null !== $e) {
-            throw new RuntimeException(sprintf('Might be related to the last exception: %s %s', get_class($e), $e->getTraceAsString()));
+            throw new RuntimeException(sprintf('Might be related to the last exception: %s: %s Use -vvv for additional stack trace info', get_class($e), $e->getMessage()), 0, $e);
```

Comment:
@zuk3975 we talked about this one, I improved it a bit The duplicate messages were because the exception was never cleaned, so it repeated this display for each following scenarios

I also reduced the message so that it focuses on last error, the stack trace can already be displayed use verbosity parameter so I'm not sure we should display it all the time